src/prefect/hello_world.py

```diff
@@ -18,6 +18,11 @@
     ------------------------------------------
     $ prefect create project 'default'
     $ prefect register --project default -m prefect.hello_world
+
+
+    Run this flow with the Prefect backend and agent
```

Comment:
is there a way to make it clearer that an agent must be running for this to work?

Reply:
Well, `--watch` is included which means after 15 seconds it'll tell you if your agent is not running

Reply:
that's how I found out 😄